api/descriptions/tags/tag_cve_post.md

```diff
@@ -1,6 +1,10 @@
 Sets a tag to a vulnerability based on Common Vulnerability and Exposures (CVE) ID, package, and resource.
-When you apply a tag to a vulnerability only on a package, the tag applies to the vulnerability in all the resources related to it.
 
+**Consider the following scenarios**: 
+  - When you apply a tag to a vulnerability only on a package, the tag applies to the vulnerability in all the resources related to it.
+  - When you apply a tag only to a vulnerability, the tag applies to the vulnerability in all the packages and resources related to it.
+  - When you apply a tag to a vulnerability on a resource type, specify the scope of the resources using either a wildcard "*" or resource names.
```

Comment:
@MorShabiPAN : Added this point to explain the relationship for resource and resource types. Followed by examples below. Please review.

Reply:
looks good